apps/docs/src/importing/trakt.md

```
@@ -15,3 +20,16 @@ their ratings, history, comments and lists. A few points to note:
+## List
+
+All items from a public Trakt list can be imported and added to the collection of your
+choice.
+
+1. Enter a URL for the list to import. For example:
+
+```txt
+https://trakt.tv/users/felix66/lists/trakt-movie-the-new-york-times-guide-to-the-best-1-000-movies-ever-made?sort=rank,asc`.
+```
+
+2. Select the collection you want to import the items into.
```

markdownlint-cli2 (0.17.2)

Ordered list item prefix
Expected: 1; Actual: 2; Style: 1/1/1

(MD029, ol-prefix)